aips/aip-x-disable-object-burn.md

```diff
@@ -0,0 +1,139 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Disable Object Burn
+author: gregnazario (greg@aptoslabs.com)
+discussions-to (*optional): <a url pointing to the official discussion thread>
+Status: Draft
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedbacks and reviews>
+type: Standard (Framework)
+created: 8/28/2024
+updated (*optional):
+requires (*optional):
+---
+
+# AIP-X - Disable Safe Burn of User-Owned Objects
+
+## Summary
+
+Safe burn was originally described in [AIP-45](./aip-45.md), which allowed users to tombstone their objects and remove
+direct linkage of an undeletable, also known as *soulbound*, object to an account. This AIP seeks to disable safe
+object burn, as it caused extra complexity, and sometimes unexpected consequences. Owners of burned objects will still
+be able to unburn and recover their tombstoned objects.
```

Comment:
especially with the desire to create untransferable objects and the notion of ownership which becomes muddy when the burn address manages the object but the object is still owned by another address.